azblob/zt_sas_identity_test.go

```diff
@@ -0,0 +1,85 @@
+package azblob_test
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/Azure/azure-storage-blob-go/azblob"
+	chk "gopkg.in/check.v1"
+	"strings"
+	"time"
+)
+
+func (s *aztestsSuite) TestIdentitySASUsage(c *chk.C) {
```

Comment:
please consider adding some comments + grouping lines together to improve readability.